endpoints/openrtb2/auction.go

```diff
@@ -1575,18 +1575,21 @@ func (deps *endpointDeps) parseBidExt(req *openrtb_ext.RequestWrapper) error {
 }
 
 func (deps *endpointDeps) validateAliases(aliases map[string]string) error {
-	for alias, coreBidder := range aliases {
-		if _, isCoreBidderDisabled := deps.disabledBidders[coreBidder]; isCoreBidderDisabled {
-			return fmt.Errorf("request.ext.prebid.aliases.%s refers to disabled bidder: %s", alias, coreBidder)
+	for alias, bidderName := range aliases {
+		normalisedBidderName, _ := openrtb_ext.NormalizeBidderName(bidderName)
```

Comment:
should check if `normalisedBidderName` is returned from `openrtb_ext.NormalizeBidderName`

Reply:
This is being checked already by `deps.bidderMap[coreBidderName]` and the error - `request.ext.prebid.aliases.test refers to unknown bidder: unknown` will be returned and that is what the error should be.